components/ai_chat/resources/page/components/prompt_auto_suggestion/index.tsx

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import styles from './style.module.scss'
+import Icon from '@brave/leo/react/icon'
+import Button from '@brave/leo/react/button'
+import { getLocale } from '$web-common/locale'
+
+interface PromptAutoSuggestionProps {
+  onEnable?: () => void
+  onDismiss?: () => void
+}
+
+function PromptAutoSuggestion (props: PromptAutoSuggestionProps) {
+  return (
+    <div className={styles.box}>
+      <i><Icon name="product-brave-ai" /></i>
```

Comment:
nit: not sure the `<i>` is neccessary, the className and the style rules can go on the `<Icon>` directly

Reply:
removed